octoprint_mrbeam/static/js/laserhead_changed.js

```diff
@@ -0,0 +1,60 @@
+$(function () {
+    function LaserheadChangedViewModel(params) {
+        let self = this;
+        window.mrbeam.viewModels["LaserheadChangedViewModel"] = self;
+
+        self.settings = params[0];
+        self.loginState = params[1];
+
+        self.onUserLoggedIn = function () {
+            if (
+                self.loginState.currentUser?.()?.active
```

Comment:
looks interesting, could `currentUser` not exist? Do you need the extra `?.` ?

Reply:
Yes I think that happens in the scenario when the session expires